src/js/components/DataTable/__tests__/DataTable-test.js

```diff
@@ -1116,4 +1116,122 @@ describe('DataTable', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
```

Comment:
Would it be worthwhile to add a test checking that `value` is being stored appropriately in these scenarios? I.e. no data duplications, only contains unique items, etc.?

I don't see anything that would cause it in this work, but wondering from a defensive standpoint?

Reply:
It's a good point. the `selected` value will hold the primaryKeys. However, it's up to the grommet user to denote which column holds the data that will serve as the primaryKey. Docs: https://v2.grommet.io/datatable#columns

Because we can't anticipate the data structure provided by the caller, it would be on them to provide the information around which column has primary data. However, I've added a check to my test to make sure onSelect has been called with the desired values.

Reply:
Nice. The onSelect assertion is the right approach and is the check I was hoping for. Thanks.